deployer/utils.py

```diff
@@ -11,16 +11,13 @@
 yaml = YAML(typ="safe", pure=True)
 
 
-def check_file_exists(filepath):
-    """Check a filepath exists, raise an error if not. This function is to be used for
+def assert_file_exists(filepath):
+    """Assert a filepath exists, raise an error if not. This function is to be used for
     files that *absolutely have to exist* in order to successfully complete deployment,
     such as, files listed in the `helm_chart_values_file` key in the `cluster.yaml` file
 
     Args:
-        filepath (str): Absolute path to the file that is to be checked for existence
-
-    Returns:
-        bool: Returns True if the function has not raised an error
+        filepath (str): Absolute path to the file that is to be asserted for existence
     """
     if not os.path.exists(filepath):
```

Comment:
I think `os.path.exists` is a bug we _maybe_ never would have noticed, because it would accept a directory path as well. Since the function is named `assert_file_exists` rather than `assert_path_exists` or similar.

```suggestion
    if not os.path.isfile(filepath):
```